examples/lambda/function.yaml

```
@@ -42,4 +42,147 @@ spec:
             "Sid": ""
           }
         ]
-      }
\ No newline at end of file
+      }
+---
+
+
+
+apiVersion: lambda.aws.upbound.io/v1beta1
+kind: Function
+metadata:
+  annotations:
+    meta.upbound.io/example-id: connect/v1beta1/function
```

Could you please change the group name?
```suggestion
    meta.upbound.io/example-id: lambda/v1beta1/function
```